docs/using_lagoon/active_standby.md

```diff
@@ -52,10 +53,14 @@ production_routes:
 
 > Note: Any routes that are under the section `environments.<branch>.routes` will not be moved as part of active/standby, these routes will always be attached to the environment as defined. Ensure that if you do need a specific route to be migrated during an active/standby switch, that you remove them from the `environments` section and place them under the `production_routes` section specific to if it should be an `active` or `standby` route.
 
-## Triggering a switch event
+## Triggering the active/standby switch
+### via the UI
```

Comment:
Screenshot?

Reply:
The current button is very much an interim feature while we wait to get some UX over it to determine the best way to implement the switch.

Reply:
Got it.